runeq/resources/project.py

```diff
@@ -50,7 +356,6 @@ def __init__(
             **attributes: Other attributes associated with the project
```

Comment:
Can you add a description here for `cohorts`. Also it might be easier to spot differences like this if the order of the Args descriptions was the same as the instantiation.